src/pc/controller/controller_keyboard.c

```diff
@@ -89,15 +91,24 @@ static void keyboard_read(OSContPad *pad) {
     pad->button |= keyboard_buttons_down;
     if ((keyboard_buttons_down & 0x30000) == 0x10000) {
         pad->stick_x = -128;
+        if (keyboard_buttons_down & 0x100000) pad->stick_x = -46;
```

Comment:
Protip: You can just do
```
const s16 sens = (keyboard_buttons_down & 0x100000) ? 46 : 128;
```
and then replace all the 128s and -128s with `sens` and `-sens` instead of checking in every button handler.

Reply:
changed, ty!